bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/AudioWaveUtils.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.audio;
```

Comment:
Can you move this calls to the `org.openhab.core.audio.utils` package.

Reply:
Oh I missed this package. I feel stupid again. Thanks !